File: php/utility/MakeContext.php
<?php
declare(strict_types=1);

// GameOfThronesQuotes SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class GameOfThronesQuotesMakeContext
{
    public static function call(array $ctxmap, ?GameOfThronesQuotesContext $basectx): GameOfThronesQuotesContext
    {
        return new GameOfThronesQuotesContext($ctxmap, $basectx);
    }
}

File: .sdk/tm/rb/utility/make_context.rb
# GameOfThronesQuotes SDK utility: make_context
require_relative '../core/context'
module GameOfThronesQuotesUtilities
  MakeContext = ->(ctxmap, basectx) {
    GameOfThronesQuotesContext.new(ctxmap, basectx)
  }
end
